src/workerd/api/http-test.js

```diff
@@ -90,17 +90,17 @@ export const inspect = {
     assert.strictEqual(util.inspect(url),
 `URL {
   searchParams: URLSearchParams(3) { 'a' => '1', 'a' => '2', 'b' => '3' },
-  hash: '',
-  search: '?a=1&a=2&b=3',
-  pathname: '/path',
-  port: '8787',
-  hostname: 'placeholder',
-  host: 'placeholder:8787',
-  password: 'pass',
-  username: 'user',
-  protocol: 'http:',
+  origin: 'http://placeholder:8787',
```

Comment:
Note for reviewers: Using `SetAccessorProperty` changes the iteration order of these properties... doh!